src/content/configuration/externals.md

```diff
@@ -144,26 +145,89 @@ This syntax is used to describe all the possible ways that an external library c
 
 `function (context, request, callback)`
 
-It might be useful to define your own function to control the behavior of what you want to externalize from webpack. [webpack-node-externals](https://www.npmjs.com/package/webpack-node-externals), for example, excludes all modules from the `node_modules` directory and provides some options too, for example, whitelist packages.
+It might be useful to define your own function to control the behavior of what you want to externalize from webpack. [webpack-node-externals](https://www.npmjs.com/package/webpack-node-externals), for example, excludes all modules from the `node_modules` directory and provides some options to, for example, whitelist packages.
 
-It basically comes down to this:
+The function receives three parameters:
+
+- `context` (`string`): The directory of the file which contains the import.
+- `request` (`string`): The import path being requested.
+- `callback` (`Function`): Callback function used to indicate how the module should be externalized.
+
+The callback function takes three parameters:
+
+- `err` (`Error`): Used to indicate if there has been an error while externalizing the import. If there is an error, this should be the only parameter used.
+- `result` (`string | string[] | Object`): Describes the external module. Can accept a string in the format `${type} ${path}`, or one of the other standard external formats ([string](#string), [string[]](#string-1), or [Object](#object))
```

Comment:
https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types

```suggestion
- `result` (`string` `[string]` `object`): Describes the external module. Can accept a string in the format `${type} ${path}`, or one of the other standard external formats ([`string`](#string), [`[string]`](#string-1), or [`object`](#object))
```